### PR TITLE
Fix padding for region control

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard4**
## Summary

Fixes paddings on region select

The removed grid properties did not have any effect, elsewhere the display is set to flex.

### CODE REVIEW

File: src/components/chartRegionControls/chartregioncontrols.module.scss [hunk 1]
```diff
@@ -5,7 +5,12 @@
   margin: 0 0 1rem;
   max-width: 100%;
   overflow: hidden;
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: auto;
```
Comment (VWSCoronaDashboard8) [inline src/components/chartRegionControls/chartregioncontrols.module.scss:10]: By fixing the padding you also altered the design. So this PR / title is a bit misleading. Why did you remove the grid layout? It made the buttons equal size.

Comment (VWSCoronaDashboard4) [reply src/components/chartRegionControls/chartregioncontrols.module.scss:10]: The constrained width in this case was giving troubles. Text in the labels ran too far to the borders: hence why the paddings were wrong.

After discussing with design, the same layout solution as the chart time controls was wanted.

**Review by VWSCoronaDashboard8 — APPROVED** [review]
